src/components/StoriesList/StoriesList.module.scss

```diff
@@ -0,0 +1,58 @@
+@import "../../styles/variables";
+
+.stories_list {
+  display: flex;
+  justify-content: space-between;
+  max-width: 1000px;
+  margin: 0 auto;
+
+  li {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    background: $dark-blue;
+    border-radius: 24px;
+    width: 400px;
+    margin: 20px;
+    padding: 20px;
+  }
+
+  a {
```

Comment:
Why not use the Button mixin?

Reply:
Because your button mixin clamps the buttons to be about 70px square.